internal/cephfs/util.go

```diff
@@ -149,17 +135,8 @@ func genSnapFromOptions(ctx context.Context, req *csi.CreateSnapshotRequest) (sn
 	return cephfsSnap, nil
 }
 
-func parseTime(ctx context.Context, createTime string) (*timestamp.Timestamp, error) {
-	tm := &timestamp.Timestamp{}
-	layout := "2006-01-02 15:04:05.000000"
-	// TODO currently parsing of timestamp to time.ANSIC generate from ceph fs is failing
-	var t time.Time
-	t, err := time.Parse(layout, createTime)
-	if err != nil {
-		util.ErrorLog(ctx, "failed to parse time %s %v", createTime, err)
-		return tm, err
-	}
-	tm, err = ptypes.TimestampProto(t)
+func parseTime(ctx context.Context, createTime time.Time) (*timestamp.Timestamp, error) {
```

Comment:
This function should probably have a unit test. No need to add it now, but maybe it should be reported as an Issue.

Reply:
sure I will create an issue to ad unit testing for the functions which don't make any call to the ceph cluster.